packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -36,6 +37,11 @@ export abstract class SharedObject<TEvent extends ISharedObjectEvents = ISharedO
     public get IChannel() { return this; }
     public get IFluidLoadable() { return this; }
 
+    private readonly _serializer: IFluidSerializer;
+    public get serializer(): IFluidSerializer {
```

Comment:
i think the class should implment IProvideFluidSerializer an expose this via an IFluidSerializer property

Reply:
just saw you comment. yeah. i think you might be right about this interface. does it even need to be in core interfaces? core interfaces is really about things that will cross the host-loader-runtime boundaries. seem like this could move fully into the runtime

Reply:
it's not something we've realized, but potentially we could want dds hosted differently than they are now. the thing that comes to mind is using them independently against some service. i don't think that really changes anything here, as regardless the dss will need a serializer, which i think can be defined in sharedObject base